ts/IdentificationModal.tsx

```diff
@@ -46,9 +46,7 @@ type State = {
 const contextualHelp = {
   title: I18n.t("pin_login.unlock_screen.help.title"),
   body: () => (
-    <Markdown lazyOptions={{ lazy: true }}>
-      {I18n.t("pin_login.unlock_screen.help.content")}
-    </Markdown>
+    <MarkdownViewer markdown={I18n.t("pin_login.unlock_screen.help.content")} />
```

Comment:
couple of points here:

* let's keep the name of the component as before (i.e. `Markdown`) the `Viewer` suffix doesn't seem to add anything
* the children based syntax feels much more intuitive than passing the markdown via a prop - any reason to not keep it?